src/index.ts

```diff
@@ -0,0 +1,17 @@
+import express from "express";
+import cors from 'cors';
```

Comment:
Although it is convenient to call it from browser when developing, we wouldn't do that in the real life, so cors is not needed. Please, remove it in the future.

Reply:
Done